lte/cloud/go/services/lte/servicers/indexer_servicer_test.go

```diff
@@ -111,29 +111,21 @@ func seedNetwork(t *testing.T, networkID string) {
 }
 
 func seedGateway(t *testing.T, networkID string, gatewayID string, hwID string) {
-	_, err := configurator.CreateEntity(
-		networkID,
-		configurator.NetworkEntity{
-			Type:         orc8r.MagmadGatewayType,
-			Key:          gatewayID,
-			Config:       &models2.MagmadGatewayConfigs{},
-			PhysicalID:   hwID,
-			Associations: []storage.TypeAndKey{{Type: orc8r.UpgradeTierEntityType, Key: "t0"}},
-		},
-		serdes.Entity,
-	)
+	_, err := configurator.CreateEntity(context2.Background(), networkID, configurator.NetworkEntity{
```

Comment:
Not quite the desired formatting but also not huge deal since we're trying to automate this